src/main/java/step5/domain/LottoNumber.java

```diff
@@ -0,0 +1,46 @@
+package step5.domain;
+
+import java.util.Objects;
+
+public class LottoNumber {
+
+    private static final Integer MIN = 1;
+    private static final Integer MAX = 45;
+    private final Integer lottoNumber;
```

Comment:
```suggestion
    private static final Integer MAX = 45;
    
    private final Integer lottoNumber;
```

클래스 변수와 인스턴스 변수 사이에 공백을 추가한다면 가독성이 상승합니다!